### PR TITLE
Add the theme name to the injection div

### PR DESCRIPTION
**Pull request by samelhusseini**
## The basics
- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Fixes https://github.com/google/blockly/issues/3407

### Proposed Changes

Each theme now has a name.
Set the name of the theme as a class onto the injection div. 

### Reason for Changes

So we can use CSS to customize a theme further.

### Test Coverage

Tested playground.
Ran build.

Tested on:
* Desktop Chrome

### CODE REVIEW

File: tests/mocha/trashcan_test.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
  */
 
 suite("Trashcan", function() {
-  var themeManager = new Blockly.ThemeManager(Blockly.Themes.Classic);
+  var themeManager = new Blockly.ThemeManager(null, Blockly.Themes.Classic);
```
Comment (rachel-fenichel) [inline tests/mocha/trashcan_test.js]: Not allowed to pass null here.

File: core/theme/dark.js [hunk 2]
```diff
@@ -115,7 +116,7 @@ Blockly.Themes.Dark.categoryStyles = {
 
 // This style is still being fleshed out and may change.
 Blockly.Themes.Dark =
-    new Blockly.Theme(Blockly.Themes.Dark.defaultBlockStyles,
+    new Blockly.Theme('dark', Blockly.Themes.Dark.defaultBlockStyles,
```
Comment (rachel-fenichel) [inline core/theme/dark.js]: `.dark` is very vague.  I suggest that we either auto-append `'Theme'` to their name, or add it just for our built-in themes.

Comment (samelhusseini) [reply core/theme/dark.js]: Fair enough.

File: tests/mocha/trashcan_test.js (hunk 1 above)
Comment (samelhusseini) [reply tests/mocha/trashcan_test.js]: Fixed.

File: core/theme/dark.js (hunk 2 above)
Comment (samelhusseini) [reply core/theme/dark.js]: Done.

**Comment by BeksOmega** [comment]
This change should be tagged for doc updates, as Blockly.Theme is a public constructor. The section on [creating a theme](https://developers.google.com/blockly/guides/configure/web/themes#create_a_theme) needs to be updated to include the name parameter.

Also important for upgrade notes (if you're doing that for the next release).

**Pull request merged**